### DISCUSSION TITLE
Other Vulnerabilities 解题讨论

### DISCUSSION THREAD

**Discussion by wumingzhilian** [HUSTSeclab/software-security-dojo#24] [General]
我们在这里讨论 Other Vulnerabilities 模块中遇到的问题

**Comment by zllancc** [comment]
![图片](https://github.com/HUSTSeclab/software-security-dojo/assets/150664801/5fa2d5c5-b4d1-4ce2-8d7a-b7450a991ce8)
这里的buf大小是不是写错了，这样就可以用bufferoverflow做这道题

**Reply by wumingzhilian** [reply]
是的，buf有误，建议经过测试后提交pr

**Comment by RutingZhang0429** [comment]
1.1通过判断条件后怎么查看/flag，用cat的话提示permission denied

**Reply by zllancc** [reply]
我的也是
